test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+--bail
+--ui tdd
+--require esm
```

Comment:
Because the test file is considered commonjs, I have to include this runtime transpiler.